recipes/pocl/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pocl" %}
+{% set version = "0.13" %}
+{% set sha256 = "a17f37d8f26819c0c8efc6de2b57f67a0c8a81514fc9cd5005434e49d67499f9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://portablecl.org/downloads/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+  # Put binary paths built in pocl into a separate compilation unit
+  # so that conda can successfully change them. (If they're not in
+  # a separate compilation unit, the compiler has a habit of evaluating
+  # strlen() and such at compile time.)
+  - paths-in-separate-compilation-unit.patch
+
+  # http://stackoverflow.com/a/39190281
+  # https://llvm.org/bugs/show_bug.cgi?id=29162
+  - work-around-skylake-codegen.patch
+
+  # Disable unneeded macro workaround
+  - fix-macros.patch
+
+  # https://github.com/pocl/pocl/issues/419
+  - 0001-Fix-clEnqueueFillBuffer-memory-leak.patch
+
+build:
+  number: 4
```

Comment:
I started this at 4 because I have had a version of this in [my own channel](https://anaconda.org/inducer/pocl/files), and I want to make sure that the folks who currently use that package have a smooth upgrade path. I'm happy to revert to zero if you think that starting at zero is the overriding concern, but overall I thought, "it's just a number", and figured obvious upgrade behavior from the out-of-tree package was more important than clean numbering. Let me know if you feel strongly about this, I'll revert if you prefer.